src/config/authd-config.h

```diff
@@ -51,6 +51,15 @@ typedef struct authd_config_t {
     bool worker_node;
 } authd_config_t;
 
+/**
+ * @brief It converts a time string with the format <time><unit>, where the unit could be
+ *        d (days), h (hours), m (minutes), or s (seconds), to a representation in seconds saved
```

Comment:
Maybe we could add that, **<unit>** is optional and if no **<unit>** is provided, the time is assumed as seconds by default.

Reply:
Fixed.